internal/proxy/meta_cache.go

```diff
@@ -350,19 +350,12 @@ func (m *MetaCache) GetPartitionID(ctx context.Context, collectionName string, p
 }
 
 func (m *MetaCache) GetPartitions(ctx context.Context, collectionName string) (map[string]typeutil.UniqueID, error) {
-	_, err := m.GetCollectionID(ctx, collectionName)
+	collInfo, err := m.GetCollectionInfo(ctx, collectionName)
 	if err != nil {
 		return nil, err
 	}
 
 	m.mu.RLock()
```

Comment:
It seems that `lock` is not necessary here.

Reply:
updated